root/etc/s6-overlay/s6-rc.d/init-flexget-config/run

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/with-contenv bash
+# shellcheck shell=bash
+
+mkdir -p \
+    /config/.flexget \
+    /data
+
+# clean up config-lock from unclean shutdown
+if [[ -f "/config/.flexget/.config-lock" ]]; then
+    rm -rf "/config/.flexget/.config-lock"
+fi
+
+if [[ ! -f "/config/.flexget/config.yml" ]]; then
+    cp /default/config.yml /config/.flexget/config.yml
+fi
+
+if [[ -n "${FG_WEBUI_PASSWORD}" ]]; then
+    if ! flexget --loglevel error web passwd "${FG_WEBUI_PASSWORD}" | tee /dev/stderr | grep -q 'Updated password'; then
+        exit 1
```

Comment:
should this exit 1? or should it sleep infinity?

Reply:
Yes, but also the FG_WEBUI_PASSWORD should be listed as optional (as you can do it via the CLI and the webui is optional anyway).

Reply:
So if the if statement here fails we're exiting the script and not doing the permission ownership at the bottom?

Reply:
Sorry, that was ambiguous. Yes it should have been a sleep. Exiting 1 would halt the init too, but would also cause a loop if the container is set to restart.